scanner/decisions/0004-scannerv4-read-vulneralbilities.md

```diff
@@ -0,0 +1,24 @@
+# 0004 - Scanner V4 NVD CVSS enrichment data updating pipeline
+
+- **Author(s):** Yi Li
+- **Created:** [2023-09-11]
+
+## Context
+
+We are in the process of transitioning the StackRox Scanner to utilize ClairCore as its primary scanning engine. Our objective is twofold: firstly, to allow Central to fetch NVD CVSS data, and secondly, to ensure the Scanner V4 seamlessly connecting with Central, triggering data updates and enriching vulnerabilities with CVSS data. Throughout this pipeline, the Claircore enricher will play a pivotal role. It will manage tasks including downloading CVSS data from Google storage, parsing the information, and updating the data accordingly for Scanner V4
+
+## Decision
+
+The CVSS Updater resides within Central, fetching CVSS data from Google storage upon request. This updater then stores the data in Central's file system. Throughout this download and save process, the Claircore CVSS enricher plays a significant role.
```

Comment:
> This updater then stores the data in Central's file system

I remember the storage team moving these definitions to the DB in another PR some time ago. Can you confirm if the decision to use the FS won't be a regression?

Reply:
> Throughout this download and save process, the Claircore CVSS enricher plays a significant role.

I apologize, but I'm having a bit of trouble understanding what you meant here. Could you please provide more information or clarification?